custom/icds_reports/reports/awc_reports.py

```diff
@@ -345,12 +345,12 @@ def get_data_for(date):
             {'age_tranche': 72},
             'nutrition_status_severely_underweight'
         )
-        wasting_moderate = exclude_records_by_age_for_column(
-            age_filters,
+        wasting_moderate = include_records_by_age_for_column(
```

Comment:
hey, could you please try debugging this function locally. What query is being prepared here?
As this looks `{'age_tranche__lt': 60}`. Will exclude kids of age 5 years. as this will become `<60`